Bump package.json version from 2.4.2 to 2.4.3 (#160)
editor-js/table: README.md

| ------------------ | -------- | ---------------------------------------- |
| `rows`             | `number` | initial number of rows. `2` by default |
| `cols`             | `number` | initial number of columns. `2` by default |

> | maxRows            | `number` | Sets the maximum number of rows allowed in the table. `Defaults` to `undefined` (no limit) |
> | `maxCols`          | `number` | Sets the maximum number of columns allowed in the table. Defaults to `undefined` (no limit) |

Reviewer: ```suggestion
| `maxRows`        | `number` | Sets the maximum number of rows allowed in the table. `undefined` by default (no limit set) |
| `maxCols`          | `number` | Sets the maximum number of columns allowed in the table. `undefined` by default (no limit set)  |
```
